wordpress-mobile/WordPress-iOS
# Fixes issue: me gravatar image not showing

Fixes #NA

Related to [this](https://github.com/wordpress-mobile/WordPressUI-iOS/pull/63) WordPressUI PR

To test:
- go to 'My Site'
- verify that the 'Me' button shows the gravatar

Note: the Podfile temporarily points to a specific commit that contains the same minor version as other dependencies. Once the new Version of wordPressUI gets merged, the other dependencies will be updated as well.

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ NA
- [x] ~~I have considered adding accessibility improvements for my changes.~~ NA
